starter-code/javascript/chronometer.js

```diff
@@ -1,39 +1,52 @@
 // Constructor
-// function Chronometer() {
+function Chronometer() {
+  this.currentTime=0;
+  this.intervalId=0;
+  
+}
 
-// }
 
-// Chronometer.prototype.startClick = function () {
+//cada 1000ms se ejecuta una suma de 1 a currentTime
+Chronometer.prototype.startClick = function () {
+    this.intervalId = setInterval(function() {
```

Comment:
lo que pasa es que no le estás dando contexto al this, entonces cuando pasa al DOM no sabe a que this.currentTime y this.setTime() estás haciendo referencia.
En este artículo se explica muy bien esto https://codeburst.io/understanding-that-and-bind-8778f779b149